website/docs/working-with-containers/registries/_insecure-registry-windows-macos.md

```diff
@@ -0,0 +1,44 @@
+#### Prerequisites
+
+- [A running Podman machine](/docs/onboarding-for-containers/installing-podman).
+
+#### Procedure
+
+1. Add your insecure registry within **<icon icon="fa-solid fa-cog" size="lg" /> Settings > Registries**.
+
+   ![Adding a custom registry](img/adding-a-custom-registry.png)
+
+1. Click "Yes" to the insecure registry warning.
+
+   ![Podman Desktop Registry Warning](img/registry-warning-insecure.png)
+
+1. SSH into the Podman Machine to edit `registries.conf`.
+
+   ```shell-session
+   $ podman machine ssh [optional-machine-name]
+   ```
+
+1. Open `registries.conf`.
+
+   ```shell-session
+   $ sudo vi /etc/containers/registries.conf
+   ```
+
+1. Add the insecure registry: Add a new `[[registry]]` section for the URL of the insecure registry you want to use. For example, if your insecure registry is located at `http://registry.example.com`, add the following lines:
+
+   ```toml
+   [[registry]]
+   location = "registry.example.com"
+   insecure = true
+   ```
+
+   If you have multiple registries, you can add one `[[registry]]` block per registry.
+
+1. Save and exit the file.
```

Comment:
I don't think we need this step as it's implied by editing the file you save and exit.

Reply:
That was already in the content. I changed as little content as possible. This might require a new pr with new scope.